.circleci/config.yml

```diff
@@ -138,7 +138,7 @@ jobs:
                 set -x
 
                 mkdir -p ./behat
-                make -j2 tfp-rabbitmq tfp-db
+                make -j2 tfp-rabbitmq tfp-db perm
```

Comment:
```suggestion
                make -j3 tfp-rabbitmq tfp-db perm
```
?

Reply:
I've changed this by doing `make perm` after `make -j2 tfp-rabbitmq tfp-db`